src/Explorer/Menus/CommandBar/MemoryTrackerComponent.tsx

```diff
@@ -1,50 +1,71 @@
-import * as React from "react";
-import { Observable, Subscription } from "knockout";
-import { MemoryUsageInfo } from "../../../Contracts/DataModels";
+import React, { FunctionComponent } from "react";
+import useSWR from "swr";
```

Comment:
>useSWR from "swr"; [](start = 7, length = 18)

While I dont think I have specific concerns with this PR, as you know, i would prefer we all talk about and agree on the framework we are adopting. We have a meeting setup early next week to discuss adopting redux/any other framework. If we want to change the philosophy and use swr where every component gets its data instead of data being passed as props, i want the same pattern to be used all across our code base. I think capacity calculator already uses redux. 

I don't want multiple patterns/frameworks dispersed in the code. Hope you agree.

Reply:
Yep I agree. This is intended to be a jumping off point for such a discussion. I do think it can be helpful to have actual usage to talk about pros/cons. Long term I certainly am against multiple approaches, but I am OK with trying out multiple approaches in production on a shorter time frame.